packages/wallets/src/features/cfd/CFDPlatformsList.tsx

```diff
@@ -13,6 +13,7 @@ const CFDPlatformsList: React.FC = () => {
     const { isMobile } = useDevice();
     const { t } = useTranslation();
     const history = useHistory();
+    const defaultText = 'Trade bigger positions with less capital on a wide range of global markets. <0>Learn more</0>';
```

Comment:
Just a suggestion can we rename it to something like `cfdDescription` or something meaningful instead of `defaultText`?

Reply:
@mitra-deriv Yeah, naming is my weakness :)